distribution/src/conf/deployment.toml

```diff
@@ -15,3 +17,144 @@ file_name = "client-truststore.jks"
+# application_json = "org.wso2.micro.integrator.core.json.JsonStreamBuilder"
+# json_badgerfish = "org.apache.axis2.json.JSONBadgerfishOMBuilder"
+# text_javascript = "org.apache.axis2.json.JSONBuilder"
+# octet_stream =  "org.wso2.carbon.relay.BinaryRelayBuilder"
+# application_binary = "org.apache.axis2.format.BinaryBuilder"
+
+# [[custom_message_formatters]]
+# content_type = text/html"
+# class = "org.wso2.carbon.relay.ExpandingMessageFormatter"
+
+# [[custom_message_builders]]
+# content_type = "text/html"
+# class = "org.wso2.carbon.relay.BinaryRelayBuilder"
+
+# [blocking.message_formatters]
+# octet_stream = "org.wso2.carbon.relay.ExpandingMessageFormatter"
+
+# [blocking.message_builders]
+# octet_stream =  "org.wso2.carbon.relay.BinaryRelayBuilder"
+
+# [[blocking.custom_message_formatters]]
+# content_type = "text/html"
+# class = "org.wso2.carbon.relay.ExpandingMessageFormatter"
+
+# [[blocking.custom_message_builders]]
+# content_type = "text/html"
+# class = "org.wso2.carbon.relay.BinaryRelayBuilder"
+
+# [transport]
+# default_content_type = "empty/content"
+
+# [transport.http]
+# socket_timeout = 180000   # timeout in milliseconds
+
+# [transport.jms]
+# listener_enable = true
+
+# [[transport.jms.listener]]
+# name = "myTopicListener"
+# parameter.initial_naming_factory = "org.apache.activemq.artemis.jndi.ActiveMQInitialContextFactory"
+# parameter.provider_url = "tcp://localhost:61616"
+# parameter.connection_factory_name = "TopicConnectionFactory"
+# parameter.connection_factory_type = "topic"
+# parameter.cache_level = "consumer"
+
+# [transport.jms]
+# sender_enable = true
+
+# [[transport.jms.sender]]
+# name = "myQueueSender"
+# parameter.initial_naming_factory = "org.apache.activemq.jndi.ActiveMQInitialContextFactory"
+# parameter.provider_url = "tcp://localhost:61616"
+# parameter.connection_factory_name = "QueueConnectionFactory"
+# parameter.connection_factory_type = "queue"
+# parameter.cache_level = "producer"
+
+# [transport.jndi.connection_factories]
+# 'connectionfactory.QueueConnectionFactory' = "amqp://admin:admin@clientID/carbon?brokerlist='tcp://localhost:5675'"
+# 'connectionfactory.TopicConnectionFactory' = "amqp://admin:admin@clientID/carbon?brokerlist='tcp://localhost:5675'"
+
+# [transport.jndi.queue]
+# StockQuotesQueue = "StockQuotesQueue"
+
+# [transport.jndi.topic]
+# MyTopic = "example.MyTopic"
+
+# [transport.rabbitmq]
+# listener_enable = true
+
+# [[transport.rabbitmq.listener]]
+# name = "AMQPConnectionFactory"
+# parameter.hostname = "localhost"
+# parameter.port = 5672
+# parameter.username = "guest"
+# parameter.password = "guest"
+
+# [transport.rabbitmq]
+# sender_enable = true
+
+# [[transport.rabbitmq.sender]]
+# name = "AMQPConnectionFactory"
+# parameter.hostname = "localhost"
+# parameter.port = 5672
+# parameter.username = "guest"
+# parameter.password = "guest"
+
+# [system.parameter]
+# 'key' = "value"
+
+# [[datasource]]
+# id = "WSO2_COORDINATION_DB"
```

Comment:
should we add the carbon datasource as well?

Reply:
I think only one data source is enough , this is just  a sample for people to refer., wdyt ?

Reply:
updated in https://github.com/wso2/micro-integrator/pull/1689/commits/c07b184479b0870e01d6735515788d4061692bd1